services/streammanager/googlesheets/googlesheetsmanager.go

```diff
@@ -226,3 +231,64 @@ func handleServiceError(err error) (statusCode int, responseMessage string) {
 	}
 	return statusCode, responseMessage
 }
+
+func clientOptions(config Config) ([]option.ClientOption, error) {
+	var credentials Credentials
+	if config.Credentials != "" {
```

Comment:
I retained the previous behaviour of the system, i.e. if `Credentials` is empty we don't return an error immediately, instead we create a `jwt.Config` with zero values. Apparently the error will come at a later stage, inside `generateOAuthClient` when we try to acquire a token